src/style.css

```diff
@@ -1 +1,155 @@
 /* Styles go here */
+/* fonts  */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap");
+/* fonts  */
+
+/* zero styles  */
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
```

Comment:
- Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your web document